README.md

```diff
@@ -7,22 +7,29 @@ A [Cookiecutter](https://cookiecutter.readthedocs.io/en/stable/) template for pr
 The following are required and recommended tools for using this cookiecutter and the LinkML project that it generates. This is all one-time setup, so if you have already done it skip to the [next section](#creating-a-new-project)!
 
   * **Python >= 3.9**
-  
-    LinkML tools are mainly written in Python, so you will need a recent Python interpreter to run this generator and to use the generated project.
 
+    LinkML tools are mainly written in Python, so you will need a recent Python interpreter to run this generator and to use the generated project.
 
   * **pipx**
-  
-    pipx is a tool for managing isolated Python-based applications. It is the recommended way to install Poetry and cruft. To install pipx follow the instructions here: https://pypa.github.io/pipx/installation/
 
+    pipx is a tool for managing isolated Python-based applications. It is the recommended way to install Poetry and cruft. To install pipx follow the instructions here: https://pypa.github.io/pipx/installation/
```

Comment:
Did something change here or is GitHub just acting weird? Whatever it is happened a couple lines up, too.

Reply:
I was surprised as well. GH-diff gets puzzled on the change from two empty lines 17/18 to one 16 (or 12/13 to 12).

Reply:
Thanks! I suspect it was something odd like that.